apps/src/netsim/netsimStorage.js

```diff
@@ -0,0 +1,154 @@
+/**
+ * Internet Simulator
+ *
+ * Copyright 2015 Code.org
+ * http://code.org/
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * @fileoverview Interface to application storage API on pegasus/dashboard
+ */
+
+/* jshint
+ funcscope: true,
+ newcap: true,
+ nonew: true,
+ shadow: false,
+ unused: true,
+
+ maxlen: 90,
+ maxparams: 3,
+ maxstatements: 200
+ */
+/* global $ */
+'use strict';
+
+/**
+ * Namespace for app storage.
+ */
+var netsimStorage = module.exports;
```

Comment:
But... but... it's kind of a "namespace" and not a "class" insofar as Javascript has those things.  You can't `new netsimStorage()`.